src/components/interaction-designer/Connection.vue

```diff
@@ -182,19 +181,22 @@ export default {
     mouseOverHandler() {
       this.line.setOptions(this.prominentOptions)
       this.activateConnection({connectionContext: this.connectionContext})
+      this.connectionDomHandle.style.zIndex = 1
       this.$emit('lineMouseIn')
     },
     mouseOutHandler() {
       if (!this.isPermanentlyActive) {
         this.line.setOptions(this.options)
         this.deactivateConnection({connectionContext: this.connectionContext})
       }
+      this.connectionDomHandle.style.zIndex = 0
       this.$emit('lineMouseOut')
     },
     clickHandler() {
       this.isPermanentlyActive = true
       this.activateConnection({connectionContext: this.connectionContext})
       this.activateBlock({blockId: null})
+      this.connectionDomHandle.style.zIndex = 1
```

Comment:
Generally, it's good practice to use orders of magnitude on `zIndex` values so that it's easier to slide others in between. Eg. 1, 10, 100, etc. then someone can easily slide something in at 11 or 99 if they need to tune the app as complexity grows and more layers are added.

Does 10 push the connection line higher than desired?

Reply:
@bzabos Using zIndex with 10 as value brings out the connection to the top like in the screenshot. 
<img width="672" alt="Screen Shot 2021-09-15 at 9 33 03 PM" src="https://user-images.githubusercontent.com/23651142/133549909-5c348c0a-6085-4064-9e2d-9e7d24b414a0.png">